Makefile

```diff
@@ -71,8 +71,7 @@ DIST_OS_TAG ?= $(BASE_OS_TAG)
 ifeq ($(BASE_OS),ubuntu)
   BASE_OS_TAG=$(BASE_OS_TAG_UBUNTU)
   BASE_IMAGE=ubuntu:$(BASE_OS_TAG_UBUNTU)
-  # Temporarily build from APT
-  DLDT_PACKAGE_URL=""
```

Comment:
We cant use s3 this is internal path. For local testing it is ok but cannot be published on github.
Try removing the empty setting and double check if it is building from ENV variable on Jenkins.
# Temporarily build from APT
  DLDT_PACKAGE_URL=""